spec/e2e/template_spec.sh

```diff
@@ -22,13 +22,13 @@ Describe './snyk-iac-rules template ./fixtures/custom-rules --rule test'
    It 'returns passing test status'
       When call ./snyk-iac-rules template ./fixtures/custom-rules --rule test
       The status should be success
-      The stderr should include 'Template rules directory'
-      The stderr should include 'Template rules/test directory'
-      The stderr should include 'Template rules/test/main.rego file'
-      The stderr should include 'Template rules/test/main_test.rego file'
-      The stderr should include 'Template rules/test/fixtures directory'
-      The stderr should include 'Template rules/test/fixtures/allowed.tf file'
-      The stderr should include 'Template rules/test/fixtures/denied.tf file'
+      The output should include 'Template rules directory'
```

Comment:
The `stderr` was from the progress bar which was writing to the error stream. Now that we've deleted it, we don't need to read the errors